### PR TITLE
[15.0][MIG] partner_time_to_pay

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

File: partner_time_to_pay/tests/test_partner_time_to_pay.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            {"name": "Bank", "type": "bank", "code": "BNK67"}
+        )
+
+        self.register_payments = self.register_payments_model.with_context(
+            active_model="account.move", active_ids=[self.invoice_id.id]
+        ).create(
+            {
+                "payment_date": self.today + relativedelta(days=10),
+                "journal_id": self.bank_journal_euro.id,
+                "payment_method_id": self.payment_method_manual_in.id,
```
Comment (Shide) [inline partner_time_to_pay/tests/test_partner_time_to_pay.py]: `payment_method_id` field doesn't exists on `account.payment.register` wizard model

**Review by Shide — CHANGES_REQUESTED** [review]
Fix the payment method manual in to fill the correct field

**Comment by rafaelbn** [comment]
Hello @Murtaza-OSI ! 😄 

Any update here? Did you see the comments?

Thanks!

**Comment by bosd** [comment]
Cool! would like to have this one on V14 as well.

**Comment by Shide** [comment]
@Murtaza-OSI are you going to continue with this pr?

**Comment by agyamuta** [comment]
> @Murtaza-OSI are you going to continue with this pr?

@Murtaza-OSI can you please fix this? Thanks.

**Comment by rafaelbn** [comment]
Hello @max3903 , is @Murtaza-OSI working on this anymore? Thank you!

**Comment by rafaelbn** [comment]
Hello @max3903 , already migrated to v14 and I ask again 😄 for v15

What do you think if we add in this module a column in invoices view lists with the date of the (last) payment which make full reconcile? Is something do you agree just in case? 😄

@Freni-OSI are you going to take care of this one too?

Thanks

**Comment by max3903** [comment]
@rafaelbn Sure, why not. Problem is that I don't have a project on 15.0 to do it. Maybe @agyamuta.

**Comment by Shide** [comment]
@Murtaza-OSI Tests are failing: https://github.com/OCA/account-invoice-reporting/actions/runs/3216983257/jobs/5259444875#step:8:109

**Comment by rafaelbn** [comment]
superseed by https://github.com/OCA/account-invoice-reporting/pull/249

**Pull request closed without merging**